docs/notebooks/dispersion/hollweg_dispersion.ipynb

```diff
@@ -0,0 +1,790 @@
+    "ax.set_xlim(0.0, 2.0)\n",
+    "ax.set_ylim(0.0, 2.0)\n",
+    "for spine in ax.spines.values():\n",
+    "    spine.set_linewidth(2)\n",
+    "ax.minorticks_on()\n",
+    "ax.tick_params(which=\"both\", labelsize=fs, width=2)\n",
+    "ax.tick_params(which=\"major\", length=10)\n",
+    "ax.tick_params(which=\"minor\", length=5)\n",
+    "ax.xaxis.set_major_locator(MultipleLocator(0.5))\n",
+    "ax.xaxis.set_minor_locator(MultipleLocator(0.1))\n",
+    "ax.yaxis.set_major_locator(MultipleLocator(0.5))\n",
+    "ax.yaxis.set_minor_locator(MultipleLocator(0.1))\n",
+    "\n",
+    "# Bellan Alfven mode\n",
+    "(p2,) = plt.plot(\n",
+    "    bellan_plt_vals[\"alfven_mode\"][\"x\"],\n",
+    "    bellan_plt_vals[\"alfven_mode\"][\"y\"],\n",
+    "    \"--\",\n",
+    "    c=slicedCM[1],\n",
+    "    linewidth=2,\n",
+    "    label=\"Alfvén\",\n",
+    ")\n",
+    "\n",
+    "# Bellan Acoustic mode\n",
+    "(p3,) = plt.plot(\n",
+    "    bellan_plt_vals[\"acoustic_mode\"][\"x\"],\n",
+    "    bellan_plt_vals[\"acoustic_mode\"][\"y\"],\n",
+    "    \"--\",\n",
+    "    c=slicedCM[2],\n",
+    "    linewidth=2,\n",
+    "    label=\"Acoustic\",\n",
+    ")\n",
+    "\n",
+    "# Hollweg Fast mode\n",
+    "(p4,) = plt.plot(\n",
+    "    hollweg_plt_vals[\"fast_mode\"][\"x\"],\n",
+    "    hollweg_plt_vals[\"fast_mode\"][\"y\"],\n",
+    "    c=slicedCM[0],\n",
+    "    linewidth=2,\n",
+    "    label=\"Fast\",\n",
+    ")\n",
+    "\n",
+    "# Hollweg Alfven mode\n",
+    "(p5,) = plt.plot(\n",
+    "    hollweg_plt_vals[\"alfven_mode\"][\"x\"],\n",
+    "    hollweg_plt_vals[\"alfven_mode\"][\"y\"],\n",
+    "    c=slicedCM[1],\n",
+    "    linewidth=2,\n",
+    "    label=\"Alfvén\",\n",
+    ")\n",
+    "\n",
+    "# Hollweg Acoustic mode\n",
+    "(p6,) = plt.plot(\n",
+    "    hollweg_plt_vals[\"acoustic_mode\"][\"x\"],\n",
+    "    hollweg_plt_vals[\"acoustic_mode\"][\"y\"],\n",
+    "    c=slicedCM[2],\n",
+    "    linewidth=2,\n",
+    "    label=\"Acoustic\",\n",
+    ")\n",
+    "\n",
+    "# annotations\n",
+    "r = np.linspace(0, 2, 200)\n",
+    "X = (r ** 2) * (np.sin(np.pi / 2 - 0.1))\n",
+    "Y = (r ** 2) * (np.cos(np.pi / 2 - 0.1))\n",
+    "plt.plot(X, Y, color=\"0.3\")\n",
+    "ax.fill_between(X, 0, Y, hatch=\"\\\\\\\\\", color=\"0.7\", alpha=0.5)\n",
+    "\n",
+    "# style legend\n",
+    "styles = [\"-\", \"--\"]\n",
+    "s_labels = [\"Hollweg\", \"Bellan\"]\n",
+    "ax2 = ax.twinx()\n",
+    "for ss, lab in enumerate(styles):\n",
+    "    ax2.plot(np.NaN, np.NaN, ls=styles[ss], label=s_labels[ss], c=\"black\")\n",
+    "ax2.get_yaxis().set_visible(False)\n",
+    "ax2.legend(fontsize=17, loc=\"center right\")\n",
+    "\n",
+    "ax.legend(handles=[p4, p1, p5, p2, p6, p3], fontsize=16, ncol=3, loc=\"upper right\")\n",
+    "plt.figtext(0.35, 0.16, \"$|\\\\theta - \\\\pi / 2| \\geq 0.1$\", rotation=5.5, fontsize=20)\n",
```

Comment:
```suggestion
    "plt.figtext(\n",
    "    1.42,\n",
    "    0.19,\n",
    "    \"$|\\\\theta - \\\\pi / 2| > 0.1 \\\\uparrow$\",\n",
    "    rotation=5.5,\n",
    "    fontsize=20,\n",
    "    transform=ax.transData,\n",
    ")\n",
```

The `transform` keyword here allows us to use data coordinates to position the text.

In this suggestion I moved the annotation to the far right of the plot area.  I think this makes the plot area a little cleaner, but this is your work so feel free to keep it where you originally had it.  The only part of this suggest I want to see added is the upward arrow in the annotation.  I do feel this greatly reduces the chances of misinterpreting the plot.

![image](https://user-images.githubusercontent.com/29869348/168700135-043f2a76-0d4c-4895-b2fd-6553d7d5eb9e.png)